exalearn-ui/src/app/components/new-topic/new-topic.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="load">
+	<mat-progress-bar *ngIf="load" color="primary" mode="indeterminate"></mat-progress-bar>
+</div>
+
+<div class="topic-page">
+	<div class="add-topic-button">
+		<div class="empty-field"></div>
+		<div class="topic-buttons">
+			<button mat-fab color="primary" (click)="addTopic()"><mat-icon>add</mat-icon></button>
```

Comment:
Translation?

Reply:
No i don't think so. It is just name of icon :)